app/assets/javascripts/components/course.cjsx

```diff
@@ -61,7 +61,7 @@ Course = React.createClass(
     user_role = if user_obj? then user_obj.role else -1
 
     if (user_role > 0 || @getCurrentUser().admin) && !@state.course.legacy && !@state.course.published
-      if CourseStore.isLoaded() && !(@state.course.submitted || @state.published)
+      if CourseStore.isLoaded() && !(@state.course.submitted || @state.published) && @state.course.type == 'ClassroomProgramCourse'
```

Comment:
The changes to course.cjsx — I'm not sure why — are causing problems only with Capybara webkit.

After adding course.type to the jbuilder, I was able to get the course_page_spec to pass (at least, didn't try others) by adding `Capybara.current_driver = :selenium`.

I'd love to get to the bottom of *why*, because that might help us figure out why some other tests only work with selenium and not headless webkit. (Webkit tests are significantly faster, so any of the slow feature specs we can move away from selenium is a win.)